centreon/tests/php/mock/Object/CentreonHost.php

```
@@ -0,0 +1,78 @@
+
+    public function setTemplates($hostId, $hostTemplates)
+    {
+
+    }
+
+    public function insertRelHostService()
+    {
+
+    }
+
+    public function getHostByAddress($host_ip, $filter)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$host_ip'. (Unused Code Rules)

(UnusedFormalParameter)